examples/mysql/getting_started_step_1/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 #[macro_use]
 extern crate diesel;
```

Comment:
Remaining `#[macro_use] extern crate diesel`

Reply:
`print_schema` will overwrite schema.rs entirely so we cannot import macro in that file manually. So I think it's fine in this case.

Reply:
I used `use diesel::prelude::*` in my `schema.rs` and it works. I think we can make `print_schema` add that line.

Reply:
@pksunkara Did you run `test_all`? It will remove any imports I think.

Reply:
No, I didn't run any schema stuff afterwards. I just wanted to try and see adding that line means I can remove the `extern crate` line. And it worked.

Reply:
As this is something that needs to change till the diesel 2.0 release anyway I would prefer to change that now: https://github.com/diesel-rs/diesel/pull/2415

Reply:
It would be also ok to just cherry pick that change to this branch 😉

Reply:
> I would prefer to change that now

How fast :) I think it's the best approach here.

Reply:
> It would be also ok to just cherry pick that change to this branch

Thanks! I cherry-picked your commit and removed all the use of `#[macro_use]` in the examples.